yarn.lock

```diff
@@ -12350,6 +12350,21 @@ __metadata:
   languageName: node
   linkType: hard
 
+"abstract-level@npm:^1.0.2, abstract-level@npm:^1.0.4":
```

Comment:
thank you @legobeat , we'll move this piece of code to a new PR, since this includes a new test flow - so there will be 2 PRs: 1 for fixing ci, the other for adding this new flow with the paste test functionality, reading the value from the database file.
Context:
https://consensys.slack.com/archives/CTQAGKY5V/p1724395549815109?thread_ts=1724321902.861359&cid=CTQAGKY5V